style.css

```diff
@@ -1,59 +1,975 @@
+
+    /* position: absolute;
+      right: 40px; */
+  }
+
+  .bar {
+    display: block;
+    width: 25px;
+    height: 3px;
+    margin: 5px;
+    margin-right: 15px;
+    -webkit-transition: all 0.3s ease;
+    transition: all 0.3s ease;
+    background-color: rgb(3, 3, 3);
+  }
+
+
+  /* .hamburger.active .bar:nth-child(2) {
+        opacity: 0;
     }
-  
     .hamburger.active .bar:nth-child(1) {
-      transform: translateY(8px) rotate(45deg);
-      background-color: #67798e;
+        transform: translateY(8px) rotate(45deg);
+        background-color: #67798e;
     }
-  
     .hamburger.active .bar:nth-child(3) {
-      transform: translateY(-8px) rotate(-45deg);
-      background-color: #67798e;
+        transform: translateY(-8px) rotate(-45deg);
+        background-color: #67798e;
     }
-  
     span.bars.active {
-      width: 10px;
-      height: 10px;
+        width: 10px;
+        height: 10px;
+    } */
+
+  .intro-section {
+    display: flex;
+    flex-direction: column;
+
+    /* margin-top: 15%; */
+    background-image: url("images/mobile_intro_bg.png");
+    background-repeat: no-repeat;
+    padding: 20px;
+    padding-top: 50px;
+    background-size: cover;
+    scroll-behavior: smooth;
+    margin-top: 13%;
+  }
+}
+
+.microverse-logo {
+  width: 100%;
+  filter: invert(70%);
+  position: relative;
+}
+
+.intro1 {
+  display: flex;
+  flex-direction: column;
+}
+
+.intro1 h1 {
+  color: orangered;
+  margin-bottom: 0;
+  font-size: 20px;
+}
+
+.intro1 h2 {
+  margin-top: 0;
+  font-size: 36px;
+  font-family: 'COCOGOOSE', sans-serif;
+  font-weight: 800;
+  background-image: url("images/grandcany.jpg");
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.intro-description {
+  padding: 20px;
+  margin: 0;
+  font-size: 15px;
+  background-color: rgb(248, 242, 242);
+}
+
+.intro-date {
+  padding: 20px;
+  margin: 0;
+  font-size: 15px;
+  font-weight: 600;
+  margin-top: 20px;
+}
+
+.main-program {
+  display: flex;
+  flex-direction: column;
+  background-image: url("images/pattern_bg.png");
+  background-color: rgb(39, 42, 50);
+  margin: 0%;
+  padding: 20px;
+  color: white;
+  align-content: space-between;
+  gap: 20px;
+  scroll-behavior: smooth;
+}
+
+#intro :-webkit-scrollbar {
+  scroll-behavior: smooth;
+}
+
+.main-text {
+  display: flex;
+  justify-content: center;
+}
+
+.main-text h3 {
+  justify-content: center;
+  border-bottom: 1px solid #ec5242;
+  padding-bottom: 10px;
+}
+
+.at-microverse {
+  text-decoration: none;
+  color: #ec5242;
+}
+
+.main-text h1 {
+  justify-content: center;
+  border-bottom: 1px solid #ec5242;
+  font-family: 'COCOGOOSE', sans-serif;
+  font-size: 25px;
+  color: #ec5242;
+  padding-bottom: 10px;
+}
+
+.main-big-box {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.main-box {
+  display: flex;
+  gap: 19px;
+  padding: 10px;
+  background: rgba(255, 255, 255, 0.1);
+}
+
+footer img {
+  /* padding-top: 10px; */
```

Comment:
- [ ] I kindly suggest that your remove commented out code from your codebase to make your work neater.